test/test_suite.py

```diff
@@ -837,3 +835,91 @@ def test_nolength_binary(self):
 @pytest.mark.skip("Spanner doesn't support quotes in table names.")
 class QuotedNameArgumentTest(_QuotedNameArgumentTest):
     pass
+
+
+@pytest.mark.skip("Spanner doesn't support IS DISTINCT FROM clause")
+class IsOrIsNotDistinctFromTest(_IsOrIsNotDistinctFromTest):
+    pass
+
+
+class OrderByLabelTest(_OrderByLabelTest):
+    @pytest.mark.skip(
+        "Spanner throws a syntax error as group by composed doesn't support."
+    )
+    def test_group_by_composed(self):
+        pass
+
+
+class CompoundSelectTest(_CompoundSelectTest):
+    @pytest.mark.skip("Spanner throws a syntax error.")
+    def test_limit_offset_selectable_in_unions(self):
+        pass
+
+    @pytest.mark.skip("Spanner throws a syntax error.")
+    def test_order_by_selectable_in_unions(self):
+        pass
+
+
+class LikeFunctionsTest(_LikeFunctionsTest):
```

Comment:
Cloud Spanner does not currently support `ESCAPE`. That's what's causing the "parenthesized expression" error. Please update the skip reason to state Spanner does not support `ESCAPE`.

Reply:
Done.